src/fsharp/creflect.fs

```diff
@@ -967,7 +967,7 @@ and ConvTyconRef cenv (tcref:TyconRef) m =
     | TProvidedTypeExtensionPoint info when not cenv.g.isInteractive && not info.IsErased -> 
         // Note, generated types are (currently) non-generic
         let tref = ExtensionTyping.GetILTypeRefOfProvidedType (info.ProvidedType, m)
-        ConvILTypeRef cenv tref
```

Comment:
There would certainly be no harm in adjusting that one for uniformity, but its harmless either way - the case is only used for the built-in primitive types like these: https://github.com/Microsoft/visualfsharp/blob/fsharp4/src/fsharp/FSharp.Core/prim-types-prelude.fsi#L73 as mentioned here https://github.com/Microsoft/visualfsharp/blob/275b832e9dd1a4bd64ed3accd218384b901be1d2/src/fsharp/tast.fs#L3681.  I think this specific case is probably only used for nativeptr if at all.